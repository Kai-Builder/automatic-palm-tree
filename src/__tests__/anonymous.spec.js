import Anonymous from '../anonymous';
import React from 'react';
import {
  shallow
} from 'enzyme';

describe('<Anonymous/>', () => {

  const requiredProps = () => ({});
  const render = (props = requiredProps()) => shallow(<Anonymous {...props}/>);

  test('component renders', () => {

    const wrapper = render();

    expect(wrapper).toMatchSnapshot();

  });

  test('component has styles', () => {

    const wrapper = render();

    expect(wrapper.props().className).toEqual('anonymous');

  });

  test('component is for anonymous users', () => {

    const wrapper = render();

    expect(wrapper.text()).toEqual('Anonymous user.');

  });

});
