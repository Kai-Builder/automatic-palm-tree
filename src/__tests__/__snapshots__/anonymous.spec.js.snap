// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anonnymous component renders 1`] = `
<section
  className=" anonymous"
>
  <div>
    Anonymous user.
  </div>
</section>
`;
